popup/calculators/taiko.ts

```diff
@@ -123,14 +102,16 @@ export const calculate = (map, mods, taikoObjects) => {
 
   const difficultyHitObjects = createDifficultyHitObjects(
     map,
-    taikoObjects,
-    clockRate
+    parsedTaikoResult,
+    clockRate,
+    convert
   )
   const sectionLength = 400 * clockRate
   let currentSectionEnd =
     Math.ceil(map.objects[0].time / sectionLength) * sectionLength
 
   difficultyHitObjects.forEach((h) => {
+    console.log(h)
```

Comment:
Can you add a linter rule for banning `console.log` statements? https://eslint.org/docs/rules/no-console

Reply:
yeah, I forgot to remove this after debugging it

Reply:
https://github.com/oamaok/ezpp/blob/2c2cc29c1a7b03dadd69cfc494999e4ea7051b8c/popup/analytics.js#L5
do we need this console.log call?

Reply:
it's handy for analytics debugging, so let's leave it.